library/std/src/io/buffered/tests.rs

```diff
@@ -832,7 +832,7 @@ fn partial_line_buffered_after_line_write() {
 }
 
 /// Test that, given a partial line that exceeds the length of
-/// LineBuffer's buffer (that is, without a trailing newline), that that
+/// LineBuffer's buffer (that is, without a trailing newline), that
 /// line is written to the inner writer
```

Comment:
`that that` is intended more often than the average doubled word. That said, it's often ok to use either one or two, and sometimes it's definitely wrong to use two.

I think i'm right for these cases...

Reply:
This double word is intentional, but this whole sentence is rather complicated so I don't blame anyone for thinking a single `that` is better. It's actually confusing enough that I am not entirely sure what this test is testing.

I'd suggest one of these rewritings:
```
/// Test that for calls to LineBuffer::write where the passed bytes do not contain
/// a newline and on their own are greater in length than the internal buffer, the
/// passed bytes are immediately written to the inner writer.
```

```
/// Test that for calls to LineBuffer::write where the passed bytes do not contain
/// a newline and would also overflow the internal buffer, the passed bytes are
/// immediately written to the inner writer.
```

Reply:
actually, i think i'll switch to the second. someone convinced me...

Reply:
I don't think those two statements are equivalent: The first one reads to me that `input.len()` is compared against `buffer.capacity()`. The second one reads to me that `input.len()` is compared against `buffer.capacity() - buffer.len()`.

Reply:
Yes. My two statements are not equivalent and I cannot tell which is intended by the original comment on account of its wording.

Reply:
Um, can you help me find someone who could venture an authoritative opinion on the intent?

Reply:
https://rust-lang.zulipchat.com/#narrow/stream/219381-t-libs/topic/Confusing.20LineBuffer.20test.20description/near/351176804